src/main/java/de/qabel/desktop/ui/about/AboutController.java

```diff
@@ -101,6 +110,38 @@ private void indexJarNames() throws ParserConfigurationException, IOException, S
         }
     }
 
+    public void openThanksPopUp() {
+        String pathThanksFile = "/files/thanks_file";
+        String contentThanksFile = readFile(pathThanksFile);
+        initializePopup();
+        setTextAreaPopup(contentThanksFile);
+        showAboutPopUp();
+    }
+
+    private void initializePopup() {
+        popupView = new AboutPopupView();
+        popupView.getViewAsync(layoutWindow.getChildren()::add);
+        popupController = (AboutPopupController) popupView.getPresenter();
+    }
+
+    private String readFile(String filePath) {
+        try (InputStream thanksFile = System.class.getResourceAsStream(filePath)){
+            return IOUtils.toString(thanksFile, "UTF-8");
+        } catch (IOException e) {
+            alert("failed to load the file" + e.getMessage(), e);
+        } catch (NullPointerException ignored){
+        }
+        return null;
```

Comment:
Exceptions over `return null`. This null will either lead to an empty popup or a NullPointerException anywhere else.
If we don't have text to show, we can't open that popup which is an illegal state for this method. So you should either throw an IllegalStateException or (preferrably) just don't catch that IOException here but catch it in the caller like
```JAVA
public void openThanksPopUp() {
    try {
        ... = readFile(...);
    } catch (IOException e) {
        alert(...);
    }
}

private String readFile(String filePath) throws IOException {
...
```